README.adoc

```diff
@@ -8,10 +8,30 @@ It includes commands to install Knative in a Kubernetes cluster, and for managin
 
 == Installation of the latest release
 
-Official binary release are available from the link:https://github.com/projectriff/riff/releases[Releases] page. See link:https://projectriff.io/docs/getting-started-with-knative-riff-on-minikube/[Getting started on Minikube] or
+=== Chocolatey
+
+To install the CLI for Windows with link:https://chocolatey.org/[Chocolatey]:
```

Comment:
Should make it clear that this will likely get you the latest *released* version of the cli (ie not master) but apart from that, lgtm

Reply:
The Chocolatey section is within `Installation of the latest release`.